### PR TITLE
Initial skeleton

### PR DESCRIPTION
**Pull request by drewvolz**
This adds a barebones react project to the site. It is by no means what it should or will look like -- it is something that will transform into a space with content, but we have to start somewhere!

olaf-today-skeleton

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
+node_modules/*
```
Comment (hawkrives) [inline .gitignore]: `node_modules/`, no star

File: index.html [hunk 2]
```diff
@@ -1,5 +1,3 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
 <title>Olaf Today</title>
-
-foo bar
```
Comment (hawkrives) [inline index.html]: Remove this file

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  border: solid 1px transparent;
+  height: 80px;
+  display: flex;
+}
+
+#app-title {
+  font-size: 60px;
+  font-weight: lighter;
+  color: white;
+  align-self: center !important;
```
Comment (hawkrives) [inline src/App.css]: remove `!important`

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+#app-logo {
+  height: 80px;
+  padding-left: 20px;
+}
+
+#app-header {
+  background-color: #f3c979;
+  border: solid 1px transparent;
+  height: 80px;
+  display: flex;
```
Comment (hawkrives) [inline src/App.css]: add `align-content: center`

Comment (drewvolz) [inline src/App.css]: Gave this a shot. Gonna not go with it.

**Comment by drewvolz** [comment]
@hawkrives thoughts?

**Pull request merged**